### ISSUE TITLE
Enabling ssl on titan 1.0 with gremlin-server errors out

### ISSUE THREAD

**Issue by jeromatron** [thinkaurelius/titan#1198]
I'm getting the following exception when trying to start gremlin-server with ssl enabled with titan 1.0:

```
6611 [main] WARN  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Enabling SSL with self-signed certificate (NOT SUITABLE FOR PRODUCTION)
Exception in thread "main" java.lang.NoClassDefFoundError: io/netty/handler/ssl/SslContextBuilder
    at org.apache.tinkerpop.gremlin.server.AbstractChannelizer.createSSLContext(AbstractChannelizer.java:207)
    at org.apache.tinkerpop.gremlin.server.AbstractChannelizer.init(AbstractChannelizer.java:114)
    at org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer.init(WebSocketChannelizer.java:55)
    at org.apache.tinkerpop.gremlin.server.GremlinServer.start(GremlinServer.java:143)
    at org.apache.tinkerpop.gremlin.server.GremlinServer.main(GremlinServer.java:291)
Caused by: java.lang.ClassNotFoundException: io.netty.handler.ssl.SslContextBuilder
    at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 5 more
```

It appears that this is because Titan only includes netty version 4.0.25 and SslContextBuilder was added to netty in version 4.0.28.  It's not a problem for the TinkerPop project because it appears to include netty 4.0.28, so it looks like a dependency problem with Titan as it's pulling in TinkerPop?

**Comment by jeromatron** [comment]
If I copy the netty-all-4.0.28-Final.jar to the titan lib directory (and mv the existing netty-all-4.0.25-Final.jar to netty-all-4.0.25-Final.jar.backup), the server starts with ssl enabled.

**Comment by spmallette** [comment]
Fixed via 2df21d5b0becc731471423018234126a781c6534 - tested ssl manually - seems to be all good.
